scripts/track_upstream_commits/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# OPTIONS:
+#   -h, --help            show this help message and exit
+#   --since SINCE         Start commit hash for git log (e.g., "bb778828e36d53a7d91a27e55109f2f45621badc").
+#   --until UNTIL         End commit hash for git log (e.g., "3ada97c109cc7ae1b451cb384a1f2cfae49c8d3e"),
+#                         it is included in the results.
+#   --crates CRATES [CRATES ...]
+#                         List of crates to track (e.g., "iota-core iota-node").
+#   --codeowner CODEOWNER
+#                         code owner of the crates (e.g., "node)
+#   --repo-url REPO_URL   The URL to the repository. Can also be a local folder.
+#   --repo-tag REPO_TAG   The tag to checkout in the repository.
+#   --version VERSION     The semantic version to filter overwrites/patches if not found in the repo-tag.
```

Comment:
Is this a copy paste leftover? If not, at least the comment is wrong.

Reply:
version is not used, it's removed now.